doc/rados/configuration/mon-config-ref.rst

```diff
@@ -423,7 +423,7 @@ by setting it in the ``[mon]`` section of the configuration file.
               is empty or identical to the last time, monitor will not send it
               to cluster log.
 :Type: Integer
-:Default: 3600
+:Default: 60.000000
```

Comment:
@falcon78921 could you take this opportunity to change the "Type" of this setting to "Float"?

Reply:
@tchaikov thanks! :)

Reply:
thank you!